src/libstd/lib.rs

```diff
@@ -320,16 +320,9 @@
 // Always use alloc_system during stage0 since we don't know if the alloc_*
 // crate the stage0 compiler will pick by default is enabled (e.g.
 // if the user has disabled jemalloc in `./configure`).
-// `force_alloc_system` is *only* intended as a workaround for local rebuilds
-// with a rustc without jemalloc.
```

Comment:
Please leave this comment in! Though if you feel like it, it could be corrected to "...for local rebuilds using a rustc with jemalloc to create a rustc without jemalloc."

Reply:
Just to clarify my previous comment, only the `// The not(stage0+msvc) gates will only last until the next stage0 bump` line should be deleted 🙂